solutions/common-shared-flows/error-handling-v1/sharedflowbundle/resources/jsc/PopulateDefaultErrorVariables.js

```diff
@@ -0,0 +1,33 @@
+// PopulateDefaultErrorVariables.js
+
+function setFault(status, phrase, code, message, url) {
+  context.setVariable("custom.error.code", code);
+  context.setVariable("custom.error.message", message);
+  context.setVariable("custom.error.url", 
+    url ? url : "https://developers.example.com");
+  context.setVariable("custom.error.status", status);
+  context.setVariable("custom.error.phrase", phrase);
+}
+
+switch (context.getVariable("fault.name")) {
+  case "access_token_expired":  
+    setFault(401, "Unauthorized", "401.98", "Unauthorized - Token Expired");
+    break;
+  case "invalid_access_token": 
+  case "InvalidAccessToken": 
+    setFault(401, "Unauthorized", "401.99", "Unauthorized - Token Invalid");
```

Comment:
As a security best practice, I've always advised against distinguishing the difference between an expired or invalid token. I've then opted for a single error message along the lines of "Token is invalid or has expired"